package.json

```diff
@@ -147,7 +147,7 @@
     "koa": "^2.13.4",
     "koa-bodyparser": "^4.3.0",
     "koa-morgan": "^1.0.1",
-    "koa-router": "12.0.0",
+    "koa-router": "12.0.1",
```

Comment:
This shouldn't be here. Please update the pull request with main again 👍

Reply:
This change is already on [main](https://github.com/instana/nodejs/blob/main/package.json#L150) I think, but you (Kathi) are right it shouldn't show up as a diff. I think this PR needs to be rebased on main? Maybe you (Athira) cherry-picked the commit from main? 🤔

Reply:
rebased to main and pushed